modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/function/ListSizeFunction.java

```diff
@@ -0,0 +1,54 @@
+package org.geotools.mbstyle.function;
```

Comment:
This should come after the header

Reply:
I should have gotten all the format issues, would appreciate one last double check.